materialize-gcs-csv/.snapshots/TestSpec

```diff
@@ -0,0 +1,98 @@
+{
+  "config_schema_json": {
+    "$schema": "http://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/estuary/connectors/materialize-gcs-csv/config",
+    "properties": {
+      "bucket": {
+        "type": "string",
+        "title": "Bucket",
+        "description": "Bucket to store materialized objects.",
+        "order": 0
+      },
+      "credentialsJson": {
+        "type": "string",
+        "title": "Service Account JSON",
+        "description": "The JSON credentials of the service account to use for authorization.",
+        "multiline": true,
+        "order": 1,
+        "secret": true
+      },
+      "uploadInterval": {
+        "type": "string",
+        "enum": [
+          "5m",
+          "15m",
+          "30m",
+          "1h"
```

Comment:
So we don't allow a value of 0 here?

Reply:
For the moment at least I'm not exposing that through the UI dropdown menu since having a huge amount of tiny files is probably not what users are going to want. But it would actually be possible to set it manually via the CLI if somebody has a reason to do so.